src/features/emails/components/EmailEditor/EmailEditorFrontend.tsx

```diff
@@ -94,6 +95,9 @@ const EmailEditorFrontend: FC<EmailEditorFrontendProps> = ({
             preserveBlank: true,
           },
         },
+        variable: {
+          class: variableToolFactory('Variable'),
```

Comment:
This hard-coded title should probably be localised before mergining :)

Reply:
OMG you are obviously right. I will fix this immediately.